librocksdb_sys/src/lib.rs

```diff
@@ -351,6 +351,14 @@ extern "C" {
                                      k: *const u8,
                                      kLen: size_t,
                                      err: *mut *mut c_char);
+    pub fn crocksdb_delete_range_cf(db: *mut DBInstance,
```

Comment:
We could live without `crocksdb_delete_range` since `delete_range_cf` is all we need for now. If we add the implementation of `crocksdb_delete_range` in this repo, we have to change it later when RocksDB adds the corresponding interfaces for it. This sounds to double the work for needless things.